python/ray/data/preprocessors/scaler.py

```diff
@@ -99,16 +99,53 @@ def __repr__(self):
 
 
 class MaxAbsScaler(Preprocessor):
-    """Scale values within columns based on the absolute max value.
+    r"""Scale each column by its absolute max value.
 
-    For each column, each value will be transformed to ``value / abs_max``,
-    where ``abs_max`` is calculated from the fitted dataset.
+    The general formula is given by
 
-    When transforming the fitted dataset, transformed values will be in the
-    range [-1, 1].
+    .. math::
+
+        x' = \frac{x}{\max{\vert x \vert}}
+
+    where :math:`x` is the column and :math:`x'` is the transformed column. If
+    :math:`\max{\vert x \vert} = 0` (i.e., the column contains all zeros), then the
+    column is unmodified.
+
+    Transformed values are always in the range :math:`[-1, 1]`.
+
+    .. note::
+        This is recommend way to scale sparse data.
```

Comment:
```suggestion
        This is the recommended way to scale sparse data.
```

Reply:
What are your thoughts on linking other scalers here? My natural reaction to reading this comment was asking "what if my data is not sparse?"

Alternatively we can follow up with a user guide.

Reply:
Fixed